safe/definitions/fields.py

```diff
@@ -2234,6 +2238,7 @@
     'key': 'affected_productivity_cost_field',
     'name': tr('Affected Productivity Cost'),
     'field_name': 'affected_productivity_cost',
+    'header_name': tr('Production Cost'),
```

Comment:
based on @fredychandra example on #3626 , it's 'Production'. CMIIW
![selection_117](https://user-images.githubusercontent.com/11134669/28665698-bf588f54-72ee-11e7-9115-3aafdabb2c11.png)